package.json

```diff
@@ -42,31 +42,47 @@
             },
             {
                 "command": "endevorexplorer.newHost",
-                "title": "New connection",
+                "title": "Add a New Datasource",
```

Comment:
Hey there. Sorry to beat a dead horse. The Endevor team will (soon, I hope) update their documentation to say "Configuration". So if we can use that throughout, that would be great.

Reply:
done in fb1e037